ipn/ipnlocal/local.go

```diff
@@ -901,7 +901,7 @@ func (b *LocalBackend) PeerCaps(src netip.Addr) tailcfg.PeerCapMap {
 	return b.peerCapsLocked(src)
 }
 
-func (b *LocalBackend) peerCapsLocked(src netip.Addr) tailcfg.PeerCapMap {
+func (b *LocalBackend) peerCapsLocked(src netip.Addr) (ret tailcfg.PeerCapMap) {
```

Comment:
unused named result. leftover defer debug probably?

Reply:
Good catch; yeah it was